js/shared.js

```diff
@@ -374,3 +376,160 @@ async function findMatchingIncipits(incipit, titleStart) {
             return Promise.resolve([])
         })
 }
+
+/**
+ * @param source A String that is either a text body or a URI to a text resource.
+ */ 
+async function getAllWitnessesOfSource(source){
+    const linesLoaded = document.querySelector("tpen-line-selector").hasAttribute("tpen-lines-loaded") ? true : false
+    if(!linesLoaded){
```

Comment:
This is the only place `linesLoaded` is used, you could probably just embed :384 here, since it is fairly self-explanatory.

Reply:
hmm curious I must have thought I would use it again.